tests/page-objects/default/tasks/tasks.wdio.page.js

```diff
@@ -1,3 +1,5 @@
+const commonPage = require('../common/common.wdio.page');
```

Comment:
Prefer using the alias, if we move the page objects, we need to update the alias' path, and the files will still work 
```suggestion
const commonPage = require('@page-objects/default/common/common.wdio.page');
```